.github/workflows/cypress_workflow.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           path: ${{ env.FTR_PATH }}
-          repository: opensearch-project/opensearch-dashboards-functional-test
+          repository: ruanyl/opensearch-dashboards-functional-test
           ref: '${{ github.base_ref }}'
```

Comment:
@ruanyl @SuZhou-Joe Do we need to set ref to `workspace` branch of ft repo to do workspace-specific test integration?

Reply:
Both should work, what about using workspace branch as the ref for our development? @ruanyl

Reply:
There have been some commits to fix workspace failed FT on the workspace branch. https://github.com/opensearch-project/opensearch-dashboards-functional-test/commits/workspace